### ISSUE TITLE
Randomy slow download speeds even though my internet speed is 100Mbps

### ISSUE THREAD

**Issue by Yilmaz4** [ytdl-org/youtube-dl#30052]
## Checklist

- [ ] I'm reporting a broken site support issue
- [x] I've verified that I'm running youtube-dl version **2021.06.06**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar bug reports including closed ones
- [x] I've read bugs section in FAQ

## Verbose log

```
[youtube] iKCMVRJGx38: Downloading webpage
[download] Destination: Coolio - Gangsta's Paradise [NFS Edit - JustResurrected Remake]-iKCMVRJGx38.m4a
[download]  10.3% of 3.86MiB at 70.67KiB/s ETA 00:50
```

## Description

I'm making a program which downloads audios from specified video's link. I'm using Python and youtube-dl. Sometimes download completes instantly. But sometimes download speed stucks at 50KiB/s or 70KiB/s. My internet speed is 100Mbps so I don't think it's a problem with my internet speed. I also tried running the program on a server but again same result, it sometimes downloads at 50KiB/s but sometimes full speed.

When I'm watching from YouTube itself, I have no problems by the way. I can watch the video at 1080p smoothly. What should I do to prevent random slow download speeds? Any ideas?

I'm running latest version of youtube-dl and latest version of Python. Below code is the code I use to download videos:
```python
import youtube_dl

ydl_opts = {
    'format': 'bestaudio/best',
    'postprocessors': [{
          'key': 'FFmpegExtractAudio',
          'preferredcodec': 'mp3',
          'preferredquality': '192',
    }],
}
with youtube_dl.YoutubeDL(ydl_opts) as ydl:
    ydl.download(["https://www.youtube.com/watch?v=iKCMVRJGx38"])
    info_dict = ydl.extract_info("https://www.youtube.com/watch?v=iKCMVRJGx38", download=False)
    video_title = info_dict.get('title', None)
```

**Comment by nicolaasjan** [comment]
This is a known issue.
See e.g. https://github.com/ytdl-org/youtube-dl/issues/29326

TL;DR;
Use the [yt-dlp fork](https://github.com/yt-dlp/yt-dlp) and all should be well again (for now at least).

**Comment by Yilmaz4** [comment]
> 
> 
> This is a known issue. See e.g. #29326
> 
> TL;DR; Use the [yt-dlp fork](https://github.com/yt-dlp/yt-dlp) and all should be well again (for now at least).

Thank you very much for your reply! I'll try to use. I hope its functions are as same as original youtube-dl.

**Comment by nicolaasjan** [comment]
> I hope its functions are as same as original youtube-dl.

Read the [Readme](https://github.com/yt-dlp/yt-dlp#readme). :)

A lot of basic functions are the same, but there are [differences](https://github.com/yt-dlp/yt-dlp#differences-in-default-behavior) as well.
